src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -67,5 +67,28 @@ public async Task<ActionResult> GetReviewStatus(string language, string packageN
 
             throw new Exception("Automatic review is not found for package " + packageName);
         }
+
+        [HttpGet]
+        public async Task<ActionResult> createApiReview(
```

Comment:
```suggestion
        public async Task<ActionResult> CreateApiReview(
```
nit: .NET usually capitalizes method names.